main/main.go

```diff
@@ -41,11 +41,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	nodeApp := app.New(nodeConfig) // Create node wrapper
 	if term.IsTerminal(int(os.Stdout.Fd())) {
 		fmt.Println(app.Header)
 	}
 
+	nodeApp, err := app.New(nodeConfig)
```

Comment:
does it matter the order wrt term?

Reply:
Yeah, this is because the node spits out logs when it starts up so we need to log it afterwards so the avalanche banner is printed first on startup